### ISSUE TITLE
Pretty printing not working by default with shadow-cljs, or with repl-pprint enabled

### ISSUE THREAD

**Issue by bpringe** [BetterThanTomorrow/calva#1259]
In a shadow-cljs project pretty printing does not work unless I change from the default pprint engine to the `calva` engine.

Setting `:repl-pprint` to `true` for the build in shadow-cljs.edn doesn't make it work either (with the default pprint engine). This seemed to work before, as noted in the docs: https://calva.io/pprint/#pprint-is-not-working-for-clojurescript. Maybe something else in my configuration is making it not work though.

 My shadow build looks like this:
 
 ```clojure
 {:target :react-native
   :init-fn my.app/init
   :output-dir "app"
   :compiler-options {:infer-externs :auto}
   :dev {:compiler-options {:closure-defines {"re_frame.trace.trace_enabled_QMARK_" true}}}
   :devtools {:autoload true
              :preloads [shadow.expo.keep-awake
                         #_re-frisk-remote.preload
                         devtools.preload]
              :repl-pprint true}}
```

@PEZ Do you find this is true for you as well? Am I missing something? If it seems to work for you just with the `repl-pprint` option enabled, then I'll need to dig further and/or create a repro project.

**Comment by bpringe** [comment]
So... I did not include cider-nrepl in my shadow repl... I just started it with `npx shadow-cljs watch app` :sweat_smile:.

I realized jacking in, which includes cider-nrepl, makes pretty printing work with the pprint engine.

I may update the docs to add a note about this. 

It seems `:repl-pprint` doesn't work, at least any more. I'm not sure why, but when I set `:repl-pprint` to true and start shadow without cider-nrepl, pprint still doesn't work (using the pprint engine). I may remove that note from our docs - though the shadow docs mention it, so it seems it should work. I'm not sure if Calva would have anything to do with it not working.
